msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/RankManagerImpl.java

```diff
@@ -362,7 +364,7 @@ private String resourceUrlEscaping(String url) {
         String leftOfAttachment = url.substring(0, attIndex);
         String rightOfAttachment = url.substring(attIndex);
 
-        String finalUrl = leftOfAttachment.concat(Validator.escapeUrl(rightOfAttachment));
+        String finalUrl = leftOfAttachment.concat(formattedText.escapeUrl(rightOfAttachment));
```

Comment:
Again, I'd leave this as Validator. FormattedText should be using URLEncoder, as per the doc in Validator.java.

Reply:
OK